client/src/components/Filter.js

```diff
@@ -1,306 +1,293 @@
+          shrink: true
+        }}
+        value={planterIdentifier}
+        onChange={e => setPlanterIdentifier(e.target.value)}
+      />
+      <GSInputLabel text='Status' />
+      <TextField
+        select
+        placeholder='e.g. 80'
+        value={status ? status : 'All'}
+        InputLabelProps={{
+          shrink: true
+        }}
+        onChange={e =>
+          setStatus(e.target.value === 'All' ? '' : e.target.value)
+        }
+      >
+        {['All', 'Planted', 'Hole dug', 'Not a tree', 'Blurry'].map(name => (
+          <MenuItem key={name} value={name}>
+            {name}
+          </MenuItem>
+        ))}
+      </TextField>
+      <GSInputLabel text='Approved' />
+      <TextField
+        select
+        value={
+          approved === undefined ? 'All' : approved === true ? 'true' : 'false'
+        }
+        InputLabelProps={{
+          shrink: true
+        }}
+        onChange={e =>
+          setApproved(
+            e.target.value === 'All'
+              ? undefined
+              : e.target.value === 'true'
+              ? true
+              : false
+          )
+        }
+      >
+        {['All', 'true', 'false'].map(name => (
+          <MenuItem key={name} value={name}>
+            {name}
+          </MenuItem>
+        ))}
+      </TextField>
+      <GSInputLabel text='Rejected' />
+      <TextField
+        select
+        value={
+          active === undefined ? 'All' : active === true ? 'false' : 'true'
+        }
+        InputLabelProps={{
+          shrink: true
+        }}
+        onChange={e =>
+          setActive(
+            e.target.value === 'All'
+              ? undefined
+              : e.target.value === 'true'
+              ? false
+              : true
+          )
+        }
+      >
+        {['All', 'false', 'true'].map(name => (
+          <MenuItem key={name} value={name}>
+            {name}
+          </MenuItem>
+        ))}
+      </TextField>
+      <GSInputLabel text='Time created' />
+      <MuiPickersUtilsProvider utils={DateFnsUtils}>
+        <Grid container justify='space-between'>
+          <KeyboardDatePicker
+            disableToolbar
+            variant='inline'
+            format='MM/dd/yyyy'
+            margin='normal'
+            id='start-date-picker'
+            label='Start Date'
+            value={dateStart}
+            onChange={handleDateStartChange}
+            KeyboardButtonProps={{
+              'aria-label': 'change date'
+            }}
+            className={classes.dateInput}
+          />
+          <KeyboardDatePicker
+            disableToolbar
+            variant='inline'
+            format='MM/dd/yyyy'
+            margin='normal'
+            id='end-date-picker'
+            label='End Date'
+            value={dateEnd}
+            onChange={handleDateEndChange}
+            KeyboardButtonProps={{
+              'aria-label': 'change date'
+            }}
+            className={classes.dateInput}
```

Comment:
Replaced the current date picker with material ui date picker. `OutlinedInput`s were replaced with `KeyboardDatePicker`